IDENTITY.md

```
@@ -17,10 +21,11 @@ Identity #2 [1]: m/44'/7778'/1'/0/0
 ...
 ```
 
-Fabric will encode each transaction using the address index field, as specified in BIP 44:
+Fabric will encode each identity address index as specified by BIP 44 for the Fabric coin type
+(not Bitcoin funds):
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)